src/features/winterStorageApplicationList/utils.ts

```diff
@@ -1,62 +1,88 @@
 import { ApplicationStatus } from '../../@types/__generated__/globalTypes';
-import { WINTER_STORAGE_APPLICATIONS } from './__generated__/WINTER_STORAGE_APPLICATIONS';
+import {
+  WINTER_STORAGE_APPLICATIONS,
+  WINTER_STORAGE_APPLICATIONS_winterStorageApplications_edges as WINTER_STORAGE_APPLICATIONS_EDGE,
+  WINTER_STORAGE_APPLICATIONS_winterStorageApplications_edges_node as WINTER_STORAGE_APPLICATIONS_NODE,
+} from './__generated__/WINTER_STORAGE_APPLICATIONS';
 
 interface WinterStorageAreaChoice {
-  winterStorageAreaName: string;
-  winterStorageArea: string;
   priority: number;
+  winterStorageArea: string;
+  winterStorageAreaName: string;
 }
 
 export type WinterStorageApplication = {
-  id: string;
-  queue: number;
-  status: ApplicationStatus;
-  createdAt: string;
-  municipality: string;
-  customerId?: string;
-  boatType?: string | null;
-  boatRegistrationNumber: string;
-  boatWidth: number;
   boatLength: number;
-  boatName: string;
   boatModel: string;
+  boatName: string;
+  boatRegistrationNumber: string;
+  boatType?: string | null;
+  boatWidth: number;
   choices: WinterStorageAreaChoice[];
+  createdAt: string;
+  customerId?: string;
+  firstName: string;
+  id: string;
+  lastName: string;
+  municipality: string;
+  queue: number;
+  status: ApplicationStatus;
 };
 
 export const getWinterStorageApplicationData = (
   data: WINTER_STORAGE_APPLICATIONS | undefined
 ): WinterStorageApplication[] => {
-  if (!data || !data.winterStorageApplications) {
-    return [];
-  }
-
-  return data.winterStorageApplications.edges.reduce<WinterStorageApplication[]>((acc, edge) => {
-    if (!edge || !edge.node) {
-      return acc;
-    }
-    const application = edge.node;
-    const applicationData: WinterStorageApplication = {
-      id: application.id,
-      queue: 0, // TODO
-      status: application.status,
-      createdAt: application.createdAt,
-      municipality: application.municipality,
-      customerId: application.customer?.id,
-      boatType: data.boatTypes?.find(({ id }) => id === application.boatType)?.name,
-      boatRegistrationNumber: application.boatRegistrationNumber,
-      boatWidth: application.boatWidth,
-      boatLength: application.boatLength,
-      boatName: application.boatName,
-      boatModel: application.boatModel,
-      choices:
-        application.winterStorageAreaChoices?.map((choice) => {
+  const boatTypes = data?.boatTypes;
+
+  return (
+    data?.winterStorageApplications?.edges.reduce<WinterStorageApplication[]>((acc, edge) => {
+      const {
+        boatLength,
+        boatModel,
+        boatName,
+        boatRegistrationNumber,
+        boatType,
+        boatWidth,
+        createdAt,
+        customer,
+        firstName,
+        id,
+        lastName,
+        municipality,
+        status,
+        winterStorageAreaChoices,
+      } = (edge as WINTER_STORAGE_APPLICATIONS_EDGE).node as WINTER_STORAGE_APPLICATIONS_NODE;
```

Comment:
```suggestion
      if (!edge?.node) return acc;
      
      const {
        boatLength,
        boatModel,
        boatName,
        boatRegistrationNumber,
        boatType,
        boatWidth,
        createdAt,
        customer,
        firstName,
        id,
        lastName,
        municipality,
        status,
        winterStorageAreaChoices,
      } = edge.node;
```

Reply:
Let's use the same style for all, so if this suggestion is what the team prefers, I can change all the utils.

Reply:
Yeah, but for me, it's a matter of avoiding `Type Assertion`.

Reply:
Changed